packages/cloudflare/env.d.ts

```diff
@@ -0,0 +1,12 @@
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      ASSETS: Fetcher;
+      __NEXT_PRIVATE_STANDALONE_CONFIG?: string;
+      SKIP_NEXT_APP_BUILD?: string;
+      [key: string]: string | Fetcher;
```

Comment:
```suggestion
      [key: string]: string;
```

is `Fetcher` necessary?

Reply:
not particularly